date_and_time/formatDate.js

```diff
@@ -0,0 +1,15 @@
+const date = new Date();
+
+function formatDate(date) {
```

Comment:
tests for your function don't pass.
https://plnkr.co/edit/gun7gTffKiQvAB6H?p=preview&preview

Reply:
Fixed it. It was problem with output year veriable. It returnded full year (2014), but not (14).